### PR TITLE
Add cmocean

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @kthyng. Do you want to be a maintainer of `cmocean` in `conda-forge`?

### CODE REVIEW

File: recipes/python-cmocean/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% set name = "cmocean" %}
+{% set version = "0.3" %}
+
+package:
+    name: python-{{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/python-cmocean/meta.yaml]: Well that spread like wildfire. 😄

**Comment by jakirkham** [comment]
LGTM. Do you want to wait or is this ready to go?

File: recipes/python-cmocean/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{% set name = "cmocean" %}
+{% set version = "0.3" %}
+
+package:
+    name: python-{{ name }}
```
Comment (jakirkham) [inline recipes/python-cmocean/meta.yaml]: 👍

**Comment by kthyng** [comment]
I don't know how to do that. So, I am happy to be a maintainer if it's easy
enough to do!

On Mon, Jun 6, 2016 at 4:03 PM, Filipe <notifications@github.com> wrote:

> Ping @kthyng <https://github.com/kthyng>. Do you want to be a maintainer
> of cmocean in conda-forge?
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/conda-forge/staged-recipes/pull/726
> Commit Summary
>
>    - Add cmocean
>
> File Changes
>
>    - *A* recipes/python-cmocean/meta.yaml
>    <https://github.com/conda-forge/staged-recipes/pull/726/files#diff-0>
>    (38)
>
> Patch Links:
>
>    - https://github.com/conda-forge/staged-recipes/pull/726.patch
>    - https://github.com/conda-forge/staged-recipes/pull/726.diff
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/726>, or mute the
> thread
> <https://github.com/notifications/unsubscribe/ADU2Bfcv_URYQQ4sDaOHrnVzksZ0ut0aks5qJIscgaJpZM4IvUE6>
> .
>



-- 
Kristen M. Thyng
Assistant Research Professor
Department of Oceanography
Texas A&M University
Eller O&M 607
http://kristenthyng.com

**Comment by ocefpaf** [comment]
> I don't know how to do that. So, I am happy to be a maintainer if it's easy
enough to do!

Basically the maintainer "job" is to merge an eventual automatic "maintenance PR" and send updates/fixes/improvements to the recipe.

Also, since you are the author of the module, it is nice to have you listed there so you can see what is being done with your work :wink:

**Pull request merged**